### PR TITLE
Build llvm backend into k

### PR DESCRIPTION
**Pull request by dwightguth**

### CODE REVIEW

**Review by traiansf — APPROVED** [review]

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by denis-bogdanas** [comment]
I don't think I have anything to do with this piece of K.

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
@traiansf can you rereview?

File: llvm-backend/pom.xml [hunk 1]
```diff
@@ -28,12 +28,46 @@
...
+              <target>
+                <mkdir dir="${project.build.directory}/build" />
+                <exec executable="cmake" dir="${project.build.directory}/build" failonerror="true">
+                  <env key="CC" value="clang-6.0" />
+                  <env key="CXX" value="clang++-6.0" />
+                  <arg value="-DCMAKE_BUILD_TYPE=${project.build.type}" />
+                  <arg value="${project.basedir}/src/main/native/llvm-backend" />
+                </exec>
+                <exec executable="make" dir="${project.build.directory}/build" failonerror="true">
+                  <arg value="-j12" />
```
Comment (traiansf) [inline llvm-backend/pom.xml:58]: Wouldn't -j12 be too hard on a machine with less cores?

File: README.md [hunk 2]
```diff
@@ -43,6 +49,20 @@ to the installation directory of the JDK (not to be mistaken with JRE).
 You can test if it works by calling `mvn -version` in a Terminal.
 This will provide the information about the JDK Maven is using, in case
 it is the wrong one.
+
+## Rust 1.28 (and Cargo)
```
Comment (traiansf) [inline README.md:53]: I wondered several times whether it would be better to have separate install instructions for each backend

**Review by traiansf — APPROVED** [review]

**Comment by ehildenb** [comment]
Jenkins: test this please

**Comment by dwightguth** [comment]
Jenkins: test this please

**Pull request merged**